NConfiguration/Serialization/SimpleTypes/Converter.cs

```diff
@@ -26,8 +27,9 @@ public static bool IsPrimitive(Type type)
 		}
 
 		private static readonly CultureInfo _ci = CultureInfo.InvariantCulture;
+	    private static readonly TimeSpanParser _timeSpanParser = new TimeSpanParser();
 
-		public static object TryCreateFunction(Type targetType)
+        public static object TryCreateFunction(Type targetType)

```

Comment:
This file uses `\t` to indent, replace indentation spaces with tabs.

Reply:
Fixed, thank you. It was different settings in VS(